tensorflow/compiler/tests/build_defs.bzl

```diff
@@ -1,12 +1,13 @@
 """Build rules for Tensorflow/XLA testing."""
 
 load("@local_config_cuda//cuda:build_defs.bzl", "cuda_is_configured")
+load("//tensorflow/compiler/tests:plugin.bzl", "plugin_device", "plugin_types")
 
 def all_backends():
+  b = ["cpu", "plugin"]
```

Comment:
I think this is already an improvement, thanks!
I would prefer that we avoid explicitly mentioning a "plugin" backend by name here. It isn't especially intuitive and we might want more than one such backend.

Could we instead have plugin.bzl contain a dictionary of (backend name, flags) entries, rather than hardcoding one plugin named "plugin"?